features/step_definitions/task_steps.rb

```diff
@@ -43,3 +43,9 @@
 Then('I should see my task list') do
   expect(page).to have_content("My Tasks")
 end
+
+Then('I should see {string} as the interval for {string}') do |interval, task_name|
+  within(:xpath, "//div[contains(@class, 'bg-white') and contains(., '#{task_name}')]") do
```

Comment:
This way of looking up the div seems odd. What "identifies" the bit we're actually looking for, better than "bg-white"?